src/Init/Tactics.lean

```diff
@@ -933,6 +933,20 @@ example : False ∨ True := by
 -/
 syntax (name := dbgTrace) "dbg_trace " str : tactic
 
+/--
+Similar to `#discr_tree_key` command, `discr_tree_key t?` prints the discrimination tree keys
+for the type of a term `t` or an identifier if `t` is specified.
+Otherwise, it prints the keys for the main goal.
+It uses the default configuration for generating keys.
```

Comment:
How often do you need this tactic, compared to just copy’n’pasting the goal? I’m a bit worried about polluting the default tactic namespace (and the auto completion, where it will affect beginners) with relatively obscure debugging tools.

Reply:
We can downstream it to batteries. @semorrison asked to put things here so perhaps she can weigh in. Either is fine with me.

Reply:
If it's useful (and not just added because we can) no complaints. But also happy to hear what @semorrison says; this isn't my particular area of expertise anyways.

Reply:
I think the command is sufficient, and we don't need the tactic.